recipes/mapd-core-cpu/build.sh

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env bash
+
+set -ex
+
+env
+
+ls -la $BUILD_PREFIX/bin
+
+# sed -i option is handled differently in Linux and OSX
+if [ $(uname) == Darwin ]; then
+    INPLACE_SED="sed -i \"\" -e"
+else
+    INPLACE_SED="sed -i"
+fi
+
+# mapd-core v 4.5.0 (or older) hardcodes /usr/bin/java. Grab
+# Calcite.cpp with a fix
+# (https://github.com/omnisci/mapd-core/pull/316) from a repo:
+wget https://raw.githubusercontent.com/omnisci/mapd-core/7c1faa09dd88d0cc735b629048f74d71baa9179f/Calcite/Calcite.cpp
```

Comment:
This should be add as `sources` in the `meta.yaml`. Preferable with a `checksum`.

Reply:
Yeah, or just have a patch here.

Reply:
Upstream promises a new release in couple of weeks that would include the fix (PR316) so that the whole block will be removed. Hence, I would prefer to apply a minimal effort here.

Reply:
OK, makes sense. We can do that in the feedstock